backend/src/test/java/ch/puzzle/okr/controller/CompletedControllerIT.java

```diff
@@ -122,4 +123,24 @@ void deleteShouldThrowExceptionWhenCompletedWithIdCantBeFound() throws Exception
                 .perform(delete("/api/v2/completed/1000").with(SecurityMockMvcRequestPostProcessors.csrf()))
                 .andExpect(MockMvcResultMatchers.status().isNotFound());
     }
+
+    @DisplayName("get() should get Completed by Objective id")
+    @Test
+    void shouldGetMetricCompletedWithId() throws Exception {
+        mvc
+                .perform(get("/api/v2/completed/1").with(SecurityMockMvcRequestPostProcessors.csrf()))
+                .andExpect(MockMvcResultMatchers.status().isOk());
+    }
+
+    @DisplayName("get() should throw exception when Completed with id cant be found")
+    @Test
+    void getShouldThrowExceptionWhenCompletedWithIdCantBeFound() throws Exception {
+        doThrow(new ResponseStatusException(HttpStatus.NOT_FOUND, "Completed not found"))
```

Comment:
You are throwing an exception using mockito and then you  just verify that this exception is thrown.
But this behavior doesnt occur while testing locally, you are just getting an 401 or a 500.
Try to just return null or smth like that from the persistenceservice and let your code handle it by itself